### PR TITLE
Fix syncing problem introduced by light forward syncing PR.

### PR DESCRIPTION
**Pull request by cheatfate**

### CODE REVIEW

File: beacon_chain/sync/sync_queue.nim [hunk 1]
```diff
@@ -139,34 +139,64 @@ proc cmp*[T](a, b: SyncRequest[T]): int =
...
+
+  ok()
+
+proc checkBlobsResponse*[T](req: SyncRequest[T],
+                            data: openArray[Slot]): Result[void, cstring] =
+  if len(data) == 0:
+    # Impossible to verify empty response.
+    return ok()
+
+  if uint64(len(data)) > (req.count * MAX_BLOBS_PER_BLOCK):
```
Comment (tersec) [inline beacon_chain/sync/sync_queue.nim]: There's `lenu64` for this pattern

Comment (cheatfate) [reply beacon_chain/sync/sync_queue.nim]: Fixed in https://github.com/status-im/nimbus-eth2/pull/6714/commits/f38ffcf4a17160b3ed44f49c4c331c96db6326c6

**Comment by agnxsh** [comment]
@cheatfate is it viable to add to a kzg verification check to `checkBlobResponse`, atleast to the first sidecar against every block? wanted a take on this

**Comment by cheatfate** [comment]
> @cheatfate is it viable to add to a kzg verification check to `checkBlobResponse`, atleast to the first sidecar against every block? wanted a take on this

This is not KZG check of blobs - its response sanity check. Actual blobs verification performs here https://github.com/status-im/nimbus-eth2/blob/unstable/beacon_chain/sync/sync_manager.nim#L354-L360

**Pull request merged**